migrations/20160111211907.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE images ALTER COLUMN file_type TYPE VARCHAR(16);
```

Comment:
To prevent future similar problems here you could also just make it `text`. In Postgres there is [no performance/storage difference](http://www.postgresql.org/docs/9.2/static/datatype-character.html) between the two types (other than a length check when using `VARCHAR(X)`), the only reason to use `VARCHAR(X)` is when you actually deliberately want to restrict the length of the field.